xds/src/main/java/io/grpc/xds/internal/security/certprovider/CertProviderSslContextProvider.java

```diff
@@ -149,22 +150,40 @@ public final void updateTrustedRoots(List<X509Certificate> trustedRoots) {
     updateSslContextWhenReady();
   }
 
+  @Override
+  public final void updateSpiffeRoots(Map<String, List<X509Certificate>> spiffeRoots) {
```

Comment:
We need to somewhere throw an error if both the flat trust bundle and the SPIFFE trust map are set.

Reply:
Did this get added somewhere?

Reply:
Leaving as is per offline discussion - a config with both `ca_certificate_file` and `spiffe_trust_bundle_map_file` is considered valid and spiffe trust bundle will be used in this case